backend/models/index.js

```diff
@@ -56,4 +56,17 @@ db.sequelize = sequelize;
 db.sequelizeReadOnly = sequelizeReadOnly;
 db.Sequelize = Sequelize;
 
+db.resetDatabase = function resetDatabase() {
+  if (config.dialect !== "sqlite") {
+    console.error(
+      `resetDatabase only supports sqlite dialect, but '${config.dialect}' found. No action is taken.`
+    );
+    return;
+  }
+  fs.renameSync(
```

Comment:
To reset a DB we do the following:

1. Rename the current sqlite DB file for the backup purposes
2. Exit the backend
3. Rely on the supervisor (`runit`) to bring the service back